ts/features/wallet/onboarding/satispay/screens/search/SearchSatispayManagerScreen.tsx

```diff
@@ -11,13 +21,31 @@ type Props = ReturnType<typeof mapDispatchToProps> &
  * This screen handles the different search state (loading, ko, error, success)
  * @constructor
  */
-const SearchSatispayManagerScreen: React.FunctionComponent<Props> = () => (
-  <View />
-);
+const SearchSatispayManagerScreen: React.FunctionComponent<Props> = props => {
+  const satispay = props.foundSatispay;
+  if (isError(satispay) && satispay.error.kind === "timeout") {
```

Comment:
Since we often check about this error
what about this?

`ts/utils/errors.ts`
```typescript
export const isTimoutError = (error: NetworkError): error is TimeoutError =>
  error.kind === "timeout";
```

Reply:
added in 271169c !